### PR TITLE
Changed all occurrences of apps to nuttx_apps.

### PR DESCRIPTION
**Pull request by FranFin**

### CODE REVIEW

File: _docs/tutorials/advanced/nuttx/debugging/index.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ In the extensions marketplace, enter "cortex", then install "Cortex-Debug". Depe
 
 ### Set up your project for debugging
 
-Open your project folder in Visual Studio Code -- this is usually the `NuttX` folder, or a subdirectory of `apps`.
+Open your project folder in Visual Studio Code -- this is usually the `NuttX` folder, or a subdirectory of `nuttx_apps`.
```
Comment (pablogs9) [inline _docs/tutorials/advanced/nuttx/debugging/index.md:156]: For Nuttx convenience he folder will be named  `apps` despite the repo renaming.

```suggestion
Open your project folder in Visual Studio Code -- this is usually the `NuttX` folder, or a subdirectory of `apps`.
```

File: _docs/tutorials/advanced/zephyr_emulator/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr host
 {% include first_application_common/config.md %}
 
 In this tutorial, we will use a UDP transport that looks for the agent on the port UDP/8888 at localhost, and focus on
-the out-of-the-box `ping_pong` application located at `firmware/zephyr_apps/apps/ping_pong`.
+the out-of-the-box `ping_pong` application located at `firmware/zephyr_apps/nuttx_apps/ping_pong`.
```
Comment (pablogs9) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:28]: this is from zephyr
```suggestion
the out-of-the-box `ping_pong` application located at `firmware/zephyr_apps/apps/ping_pong`.
```

File: _docs/tutorials/core/first_application_rtos/freertos.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Once the command is executed, a folder named `firmware` must be present in your
 
 This step is in charge, among other things, of downloading a set of micro-ROS apps for the specific platform you are
 addressing.
-In the case of FreeRTOS, these are located at `firmware/freertos_apps/apps`.
+In the case of FreeRTOS, these are located at `firmware/freertos_apps/nuttx_apps`.
```
Comment (ralph-lange) [inline _docs/tutorials/core/first_application_rtos/freertos.md:25]: These are the FreeRTOS apps.

File: _docs/tutorials/core/first_application_rtos/freertos.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ containing the two files just described.
 {% include first_application_common/config.md %}
 
 In this tutorial, we will use a Serial transport (labeled as `serial`) and focus on the out-of-the-box `ping_pong`
-application located at `firmware/freertos_apps/apps/ping_pong`. To execute this application with the chosen transport,
+application located at `firmware/freertos_apps/nuttx_apps/ping_pong`. To execute this application with the chosen transport,
```
Comment (ralph-lange) [inline _docs/tutorials/core/first_application_rtos/freertos.md:39]: These are the FreeRTOS apps.

File: _docs/tutorials/advanced/zephyr_emulator/index.md [hunk 5]
```diff
@@ -35,15 +35,15 @@ ros2 run micro_ros_setup configure_firmware.sh host_ping_pong --transport udp --
...
 
 The contents of the Zephyr app specific files can be found here:
-[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/src/main.c),
-[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/app-colcon.meta),
-[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/CMakeLists.txt)
-and [host-udp.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/host-udp.conf).
+[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/src/main.c),
+[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/app-colcon.meta),
+[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/CMakeLists.txt)
+and [host-udp.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/host-udp.conf).
```
Comment (jamoralp) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:46]: These are Zephyr apps. Link does not exist.

File: _docs/tutorials/advanced/zephyr_emulator/index.md [hunk 6]
```diff
@@ -35,15 +35,15 @@ ros2 run micro_ros_setup configure_firmware.sh host_ping_pong --transport udp --
 ```
 
 You can check the complete content of the `ping_pong` app
-[here](https://github.com/micro-ROS/zephyr_apps/tree/foxy/apps/ping_pong).
+[here](https://github.com/micro-ROS/zephyr_apps/tree/foxy/nuttx_apps/ping_pong).
```
Comment (pablogs9) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:38]: same
```suggestion
[here](https://github.com/micro-ROS/zephyr_apps/tree/foxy/apps/ping_pong).
```

File: _docs/tutorials/advanced/nuttx/debugging/index.md (hunk 1 above)
Comment (jamoralp) [inline _docs/tutorials/advanced/nuttx/debugging/index.md:156]: I believe that the folder is still named `apps` after it is downloaded, although the link has changed.

File: _docs/tutorials/core/first_application_rtos/freertos.md (hunk 3 above)
Comment (jamoralp) [inline _docs/tutorials/core/first_application_rtos/freertos.md:25]: Same as with Zephyr case

File: _docs/tutorials/advanced/zephyr_emulator/index.md [hunk 7]
```diff
@@ -35,15 +35,15 @@ ros2 run micro_ros_setup configure_firmware.sh host_ping_pong --transport udp --
...
 
 {% include first_application_common/pingpong_logic.md %}
 
 The contents of the Zephyr app specific files can be found here:
-[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/src/main.c),
-[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/app-colcon.meta),
-[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/CMakeLists.txt)
-and [host-udp.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/host-udp.conf).
+[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/src/main.c),
```
Comment (pablogs9) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:43]:
```suggestion
[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/src/main.c),
```

File: _docs/tutorials/core/first_application_rtos/freertos.md (hunk 4 above)
Comment (jamoralp) [inline _docs/tutorials/core/first_application_rtos/freertos.md:39]: Also here

File: _docs/tutorials/advanced/zephyr_emulator/index.md [hunk 8]
```diff
@@ -35,15 +35,15 @@ ros2 run micro_ros_setup configure_firmware.sh host_ping_pong --transport udp --
...
+[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/app-colcon.meta),
```
Comment (pablogs9) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:44]:
```suggestion
[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/app-colcon.meta),
```

File: _docs/tutorials/core/first_application_rtos/freertos.md [hunk 9]
```diff
@@ -45,13 +45,13 @@ ros2 run micro_ros_setup configure_firmware.sh ping_pong --transport serial --de
...
-[here](https://github.com/micro-ROS/freertos_apps/tree/foxy/apps/ping_pong).
+[here](https://github.com/micro-ROS/freertos_apps/tree/foxy/nuttx_apps/ping_pong).
 
 {% include first_application_common/pingpong_logic.md %}
 
 The contents of the FreeRTOS app specific files can be found here:
-[app.c](https://github.com/micro-ROS/freertos_apps/blob/foxy/apps/ping_pong/app.c) and
-[app-colcon.meta](https://github.com/micro-ROS/freertos_apps/blob/foxy/apps/ping_pong/app-colcon.meta).
+[app.c](https://github.com/micro-ROS/freertos_apps/blob/foxy/nuttx_apps/ping_pong/app.c) and
+[app-colcon.meta](https://github.com/micro-ROS/freertos_apps/blob/foxy/nuttx_apps/ping_pong/app-colcon.meta).
```
Comment (jamoralp) [inline _docs/tutorials/core/first_application_rtos/freertos.md:54]: Neither applicable here

File: _docs/tutorials/advanced/zephyr_emulator/index.md [hunk 10]
```diff
@@ -35,15 +35,15 @@ ros2 run micro_ros_setup configure_firmware.sh host_ping_pong --transport udp --
...
+[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/CMakeLists.txt)
```
Comment (pablogs9) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:45]:
```suggestion
[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/CMakeLists.txt)
```

File: _docs/tutorials/advanced/zephyr_emulator/index.md (hunk 5 above)
Comment (pablogs9) [inline _docs/tutorials/advanced/zephyr_emulator/index.md:46]:
```suggestion
and [host-udp.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/host-udp.conf).
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md [hunk 11]
```diff
@@ -23,23 +23,23 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr olimex-stm32-e407
 {% include first_application_common/config.md %}
 
 In this tutorial, we will use a USB transport (labeled as `serial-usb`) and focus on the out-of-the-box `ping_pong`
-application located at `firmware/zephyr_apps/apps/ping_pong`. To execute this application with the chosen transport,
+application located at `firmware/zephyr_apps/nuttx_apps/ping_pong`. To execute this application with the chosen transport,
```
Comment (jamoralp) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:26]: Not applicable here

File: _docs/tutorials/core/first_application_rtos/freertos.md (hunk 3 above)
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/freertos.md:25]:
```suggestion
In the case of FreeRTOS, these are located at `firmware/freertos_apps/apps`.
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md [hunk 12]
```diff
@@ -23,23 +23,23 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr olimex-stm32-e407
...
 
 The contents of the Zephyr app specific files can be found here:
-[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/src/main.c),
-[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/app-colcon.meta),
-[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/CMakeLists.txt)
-and [serial-usb.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/serial-usb.conf).
+[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/src/main.c),
+[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/app-colcon.meta),
+[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/CMakeLists.txt)
+and [serial-usb.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/serial-usb.conf).
```
Comment (jamoralp) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:42]: Not applicable here

File: _docs/tutorials/core/first_application_rtos/freertos.md (hunk 4 above)
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/freertos.md:39]:
```suggestion
application located at `firmware/freertos_apps/apps/ping_pong`. To execute this application with the chosen transport,
```

File: _includes/first_application_common/zephyr_common.md [hunk 13]
```diff
@@ -2,7 +2,7 @@ Once the command is executed, a folder named `firmware` must be present in your
 
 This step is in charge, among other things, of downloading a set of micro-ROS apps for the specific platform you are
 addressing.
-In the case of Zephyr, these are located at `firmware/zephyr_apps/apps`.
+In the case of Zephyr, these are located at `firmware/zephyr_apps/nuttx_apps`.
```
Comment (jamoralp) [inline _includes/first_application_common/zephyr_common.md:5]: Not applicable here

File: _docs/tutorials/core/first_application_rtos/freertos.md [hunk 14]
```diff
@@ -45,13 +45,13 @@ ros2 run micro_ros_setup configure_firmware.sh ping_pong --transport serial --de
 ```
 
 You can check the complete content of the `ping_pong` app
-[here](https://github.com/micro-ROS/freertos_apps/tree/foxy/apps/ping_pong).
+[here](https://github.com/micro-ROS/freertos_apps/tree/foxy/nuttx_apps/ping_pong).
```
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/freertos.md:48]:
```suggestion
[here](https://github.com/micro-ROS/freertos_apps/tree/foxy/apps/ping_pong).
```

File: _docs/tutorials/core/first_application_rtos/freertos.md [hunk 15]
```diff
@@ -45,13 +45,13 @@ ros2 run micro_ros_setup configure_firmware.sh ping_pong --transport serial --de
...
 
 {% include first_application_common/pingpong_logic.md %}
 
 The contents of the FreeRTOS app specific files can be found here:
-[app.c](https://github.com/micro-ROS/freertos_apps/blob/foxy/apps/ping_pong/app.c) and
-[app-colcon.meta](https://github.com/micro-ROS/freertos_apps/blob/foxy/apps/ping_pong/app-colcon.meta).
+[app.c](https://github.com/micro-ROS/freertos_apps/blob/foxy/nuttx_apps/ping_pong/app.c) and
```
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/freertos.md:53]:
```suggestion
[app.c](https://github.com/micro-ROS/freertos_apps/blob/foxy/apps/ping_pong/app.c) and
```

File: _docs/tutorials/core/first_application_rtos/freertos.md (hunk 9 above)
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/freertos.md:54]:
```suggestion
[app-colcon.meta](https://github.com/micro-ROS/freertos_apps/blob/foxy/apps/ping_pong/app-colcon.meta).
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md (hunk 11 above)
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:26]:
```suggestion
application located at `firmware/zephyr_apps/apps/ping_pong`. To execute this application with the chosen transport,
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md [hunk 16]
```diff
@@ -23,23 +23,23 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr olimex-stm32-e407
...
+application located at `firmware/zephyr_apps/nuttx_apps/ping_pong`. To execute this application with the chosen transport,
 run the configuration command above by specifying the `[APP]` and `[OPTIONS]` parameters as below:
 
 ```bash
 # Configure step with ping_pong app and serial-usb transport
 ros2 run micro_ros_setup configure_firmware.sh ping_pong --transport serial-usb
 ```
 You can check the complete content of the `ping_pong` app
-[here](https://github.com/micro-ROS/zephyr_apps/tree/foxy/apps/ping_pong).
+[here](https://github.com/micro-ROS/zephyr_apps/tree/foxy/nuttx_apps/ping_pong).
```
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:34]:
```suggestion
[here](https://github.com/micro-ROS/zephyr_apps/tree/foxy/apps/ping_pong).
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md [hunk 17]
```diff
@@ -23,23 +23,23 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr olimex-stm32-e407
...
 
 {% include first_application_common/pingpong_logic.md %}
 
 The contents of the Zephyr app specific files can be found here:
-[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/src/main.c),
-[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/app-colcon.meta),
-[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/CMakeLists.txt)
-and [serial-usb.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/serial-usb.conf).
+[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/src/main.c),
```
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:39]:
```suggestion
[main.c](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/src/main.c),
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md [hunk 18]
```diff
@@ -23,23 +23,23 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr olimex-stm32-e407
...
+[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/app-colcon.meta),
```
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:40]:
```suggestion
[app-colcon.meta](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/app-colcon.meta),
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md [hunk 19]
```diff
@@ -23,23 +23,23 @@ ros2 run micro_ros_setup create_firmware_ws.sh zephyr olimex-stm32-e407
...
+[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/nuttx_apps/ping_pong/CMakeLists.txt)
```
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:41]:
```suggestion
[CMakeLists.txt](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/CMakeLists.txt)
```

File: _docs/tutorials/core/first_application_rtos/zephyr.md (hunk 12 above)
Comment (pablogs9) [inline _docs/tutorials/core/first_application_rtos/zephyr.md:42]:
```suggestion
and [serial-usb.conf](https://github.com/micro-ROS/zephyr_apps/blob/foxy/apps/ping_pong/serial-usb.conf).
```

File: _includes/first_application_common/zephyr_common.md (hunk 13 above)
Comment (pablogs9) [inline _includes/first_application_common/zephyr_common.md:5]:
```suggestion
In the case of Zephyr, these are located at `firmware/zephyr_apps/apps`.
```

**Pull request merged**